orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/vectoroftrust/VectorOfTrust.java

```diff
@@ -0,0 +1,220 @@
+package uk.gov.di.orchestration.shared.entity.vectoroftrust;
+
+import com.google.gson.annotations.Expose;
+import org.jetbrains.annotations.NotNull;
+import software.amazon.awssdk.utils.Lazy;
+
+import java.text.MessageFormat;
+import java.util.Comparator;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static uk.gov.di.orchestration.shared.entity.vectoroftrust.VotConstants.AUTH_EQUIVALENCY_LIST;
+import static uk.gov.di.orchestration.shared.entity.vectoroftrust.VotConstants.AUTH_MEDIUM_LEGACY;
+import static uk.gov.di.orchestration.shared.entity.vectoroftrust.VotConstants.IDENT_EMPTY;
+import static uk.gov.di.orchestration.shared.entity.vectoroftrust.VotConstants.IDENT_EQUIVALENCY_LIST;
+import static uk.gov.di.orchestration.shared.entity.vectoroftrust.VotConstants.IDENT_NONE;
+
+/** A Vector of Trust represents a combined authentication and identity trust level. */
+public class VectorOfTrust implements Comparable<VectorOfTrust> {
+
+    /** AKA "Credential Trust Level" */
+    @NotNull @Expose private final VotComponent<AuthId> authComponent;
+
+    /** AKA "Level Of Confidence" */
+    @NotNull @Expose private final VotComponent<IdentId> identComponent;
```

Comment:
So why not call them this?

Reply:
I thought you had mentioned they were not particularly clear names (which I also think is was true -  "LoC" is vague and I believe I've seen CTLs assigned to a variable called "confidence". Hadn't initially appreciated how widely the terms were used.

Reply:
As above, I've changed the names back. Even if they are a little unclear probably best to be consistent given the widespread usage.